### PR TITLE
Start pyvespa quick start section

### PR DESCRIPTION
**Pull request by thigm85**
Start the pyvespa getting started section. More short guides will come once this is merged.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: en/getting-started.html [hunk 1]
```diff
@@ -79,6 +79,18 @@
...
 
+<tr>
+  <th>pyvespa</th>
+  <td>
+  <p>Get started with Vespa python API</p>
+
+
+  <ul>
+    <li><a href="pyvespa/connect-to-vespa-instance.html">Connect to a running Vespa instance</a>.
+    This short guide will show you how to connect to a pre-existing Vespa instance. We use the <a href="https://cord19.vespa.ai/">cord19</a> app as an example.
```
Comment (kkraune) [inline en/getting-started.html:90]: Simpler language:
```suggestion
  <p>Get started with Vespa python API:</p>
  <ul>
    <li><a href="pyvespa/connect-to-vespa-instance.html">Connect to a running Vespa instance</a>.
    Query a Vespa instance with python,
    using the <a href="https://cord19.vespa.ai/">cord19</a> sample app as an example.
```

File: en/pyvespa/connect-to-vespa-instance.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (kkraune) [inline en/pyvespa/connect-to-vespa-instance.md:2]:
```suggestion
# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

File: en/pyvespa/connect-to-vespa-instance.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+title: "Connect to a running Vespa instance"
```
Comment (kkraune) [inline en/pyvespa/connect-to-vespa-instance.md:3]: I think a better title is "Query Vespa from python" - or something like that, "connecting" is not that sexy and is a small part of it...

**Comment by kkraune** [comment]
Great guide! Generally I prefer a less personal writing style, with less/no "We can" / "to us" as this is technical doc and less text is most often better. Good to merge after addressing the comments

**Review by kkraune — APPROVED** [review]

File: en/pyvespa/connect-to-vespa-instance.md (hunk 3 above)
Comment (thigm85) [reply en/pyvespa/connect-to-vespa-instance.md:3]: I agree with the sexy part, but the problem is that this tutorial shows how to connect to a running Vespa instance. The query part is there to show that it works. There will be other tutorials where you deploy your own app and query it, so the differentiation between them is how you connect to the app and not the query part.

I will merge this, and we can reevaluate the title when we include the other tutorials.

**Pull request merged**